src/hamlib.mk

```diff
@@ -26,11 +27,11 @@ define $(PKG)_BUILD
      echo 'Description: $(PKG)'; \
      echo 'Requires: libusb-1.0'; \
      echo 'Cflags: -DHAMLIB_LIB'; \
-     echo 'Libs: -lhamlib -lpthread -lws2_32';) \
+     echo 'Libs: -lhamlib -lpthread -lws2_32 -lusb-1.0';) \
```

Comment:
The `Requires` above should mean you don't have to manually specify the lib (likewise for test program below).

Reply:
@tonytheodore you are right for the test file, but the configure will fail w/o the LIBS.

Without the manual LIBS='-lusb-1.0' \, hamlib will fail to build
`checking for libusb_init in -lusb-1.0... no`
here is a [gist](https://gist.github.com/dl5rcw/9485029267effe16da92386496658f17) with the complete output

If I place `LIBS='-lusb-1.0' \`, hamlib compiles fine.
If I place the `echo '' > *.pc` file to the top, hamlib does not compile - obviously the order in the *mk file does not affect the situation.

If I remove the `libusb-1.0` from test file, but keep the manual LIBS='-lusb-1.0' \ at the configure part, hamlib compiles fine.